src/main/java/com/google/sps/servlets/CalendarTokenServlet.java

```diff
@@ -7,11 +7,16 @@
 import javax.servlet.ServletException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
 
 import com.google.api.client.auth.oauth2.AuthorizationCodeFlow;
 
 @WebServlet("/token")
 public class CalendarTokenServlet extends AbstractAppEngineAuthorizationCodeServlet {
+  @Override
+  protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+    response.sendRedirect("/" + request.getParameter("origin") + ".html");
```

Comment:
Why does it always have to end with .html?

Reply:
I thought that for now we should redirect the user back to a html file